src/lib/php/Dao/SpashtDao.php

```diff
@@ -1,21 +1,12 @@
 <?php
 /*
-Copyright (C) 2008-2012 Hewlett-Packard Development Company, L.P.
-Copyright (C) 2014, Siemens AG
-Author: Johannes Najjar
-
-This program is free software; you can redistribute it and/or
-modify it under the terms of the GNU General Public License
-version 2 as published by the Free Software Foundation.
-
-This program is distributed in the hope that it will be useful,
-but WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-GNU General Public License for more details.
-
-You should have received a copy of the GNU General Public License along
-with this program; if not, write to the Free Software Foundation, Inc.,
-51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+Copyright 2019
+Author: Vivek Kumar<vvksindia@gmail.com>
+
+Copying and distribution of this file, with or without modification,
```

Comment:
Please use standard GPL-2.0-only license headers (earlier version was correct).

Reply:
updated in feat/spasht_5.0